osu.Game.Rulesets.Osu/Objects/Drawables/DrawableSpinner.cs

```diff
@@ -286,7 +286,7 @@ protected override void UpdateAfterChildren()
 
         private static readonly int score_per_tick = new SpinnerBonusTick.OsuSpinnerBonusTickJudgement().MaxNumericResult;
 
-        private int wholeSpins;
+        private int currentSpins;
```

Comment:
I find this rename of dubious value, especially so that a lot of the confusion / issue around getting these counts to match is that stable was counting in half-spins and lazer was (and continues to) count in full spins. Not really a huge deal but feels worse than before readability-wise.

Reply:
I don't know, I was confused with using the term "whole" to indicate current state of a variable, and the term "spin" alone will always indicate a full revolution in my head, I can't think about anything else.

Reply:
> was confused with using the term "whole" to indicate current state of a variable

Can you elaborate? I don't find anything confusing about that.

Reply:
When reading the name of it from inside `updateBonusScore` it feels like `wholeSpins` is the total and `spins` is just a portion, or at least that's what I understood at first glance.

Reply:
I don't necessarily read it that way, but if I were to counter with a proposal, I'd propose `completedFullSpins` or something.

Reply:
As a compromise, can we at least make it clear whether this is full spins or half spins in xmldoc? I wanna get this concern out of the way asap.